### PR TITLE
Store v1 removal in favor of Stores with generic keys

### LINKED ISSUE #118: Remove remainder of stuff specific to a Store<T> or ParsingStore<T,V>
since we are jumping directly to version 2 (without passing go :-)) it would make sense to remove the rest of the legacy interfaces/base classes that are depending on Barcode. IE this includes migrating the beta package to the normal one

### PR DESCRIPTION
**Pull request by digitalbuddha**
This PR completes the work of removing the old Store<Parsed> and replacing it with a Store<Parsed,Key>  

closes #118

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -54,7 +54,7 @@ ext {
 
     // POM file
     GROUP = "com.nytimes.android"
-    VERSION_NAME = "1.0.9-SNAPSHOT"
+    VERSION_NAME = "2.0.0-SNAPSHOT"
```
Comment (brianPlummer) [inline build.gradle]: woo hoo!

**Comment by digitalbuddha** [comment]
Who needs minor version ☺️

File: build.gradle (hunk 1 above)
Comment (PaulWoitaschek) [inline build.gradle]: Nice work guys :)

**Comment by digitalbuddha** [comment]
Thanks man! Could not have done it without your contributions.

On Feb 9, 2017 12:44 AM, "Paul Woitaschek" <notifications@github.com> wrote:

*@PaulWoitaschek* commented on this pull request.
------------------------------

In build.gradle <https://github.com/NYTimes/Store/pull/122>:

> @@ -54,7 +54,7 @@ ext {

     // POM file
     GROUP = "com.nytimes.android"
-    VERSION_NAME = "1.0.9-SNAPSHOT"
+    VERSION_NAME = "2.0.0-SNAPSHOT"

Nice work guys :)

—
You are receiving this because you modified the open/close state.

Reply to this email directly, view it on GitHub
<https://github.com/NYTimes/Store/pull/122>, or mute the thread

File: build.gradle (hunk 1 above)
Comment (pavlospt) [inline build.gradle]: Way to go!!

**Pull request merged**